### PR TITLE
Fix api daemon to run under non root user

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-bug-id: Tendrl/api#115

Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@brainfunked , @anivargi , @r0h4n , @nthomas-redhat , @shtripat , Please review

File: tendrl-api.spec [hunk 1]
```diff
@@ -1,4 +1,11 @@
-Name: tendrl-api
+%global name tendrl-api
+%global app_group %{name}
+%global app_user %{name}
+%global install_dir %{_datadir}/%{name}
+%global config_dir %{_sysconfdir}/tendrl
```
Comment (r0h4n) [inline tendrl-api.spec]: @anivargi  Isnt the config dir /etc/tendrl/api  instead of /etc/tendrl ?

**Comment by r0h4n** [comment]
@anivargi  @TimothyAsirJeyasing  Hold on with merging this, Is @mbukatov  (tendrl-ansible) aware of these changes

File: tendrl-api.spec (hunk 1 above)
Comment (anivargi) [reply tendrl-api.spec]: The config dir is correct and it was the same before, the api looks at /etc/tendrl/etc.yml

**Comment by r0h4n** [comment]
@TimothyAsir  please resolve merge conflicts

**Pull request merged**